pacman-nodejs-app/docs/pacman-nodejs-app-gke-migration.md

```diff
@@ -19,7 +19,7 @@ Follow these steps to [push the Pac-Man container image to your Google Cloud Con
 
 ## Create Kubernetes Clusters
 
-Follow [these instructions](http://localhost:6419/docs/kubernetes-cluster-gke-federation.md#create-the-kubernetes-clusters)
+Follow [these instructions](https://github.com/font/k8s-example-apps/blob/master/pacman-nodejs-app/docs/kubernetes-cluster-gke-federation.md#create-the-kubernetes-clusters)
```

Comment:
No need to have the entire link. Just use `Follow [these instructions](kubernetes-cluster-gke-federation.md#create-the-kubernetes-clusters)`